BattleTank/Game1.cs

```diff
@@ -542,7 +540,7 @@ protected override void Update(GameTime gameTime)
                             soundEffectInstance.Stop();
                             enemyTanks.Clear();
                             mines.Clear();
-                            tank2.Die();
+                            tank2.lives = 0;
```

Comment:
Czy ``lives = 0 `` na prawdę musi być ustalane tutaj i nie da rady go wciągnać do ``Tank.Die()``?

Reply:
Prosze o dodanie wpisu dla tego probemu, jesli jest to konieczne